npm/ng-packs/packages/theme-shared/src/lib/services/status-code-error-handler.service.ts

```diff
@@ -78,15 +78,16 @@ export class StatusCodeErrorHandlerService implements CustomHttpErrorHandlerServ
     switch (this.status) {
       case 401:
       case 404:
+        if (this.status === 401) {
+          this.authService.navigateToLogin();
+          return;
+        }
+
```

Comment:
move these lines to `case 401:`

Reply:
This time I need to check twice for customError case